rollup.config.mjs

```diff
@@ -17,6 +17,7 @@ const inputExists = (config) => {
   try {
     return fs.existsSync(config.input);
   } catch (e) {
+    console.error(e);
```

Comment:
Is deze log nodig? `inputExists` was ooit bedoeld om het makkelijk te maken optioneel aanwezige files te builden, als een file niet bestaat was dat niet per se een error.

Reply:
Aah ik wist niet dat dat niet per se een error is. Had het aangepast omdat ESLint zat te zeuren omdat die variabelen niet gebruikt werd. Heb het nu op een andere manier opgelost.